frontend/src/pages/Assessments.js

```diff
@@ -1,8 +1,15 @@
 import React from "react";
 import Layout from "../components/Layout";
+import PromptSpeech from "../components/PromptSpeech";
 
 const Assessments = () => {
-  return <Layout>Assessments</Layout>;
+  return (
+    <Layout>
+      <h1>Assessments</h1>
+      <p>Miniscule</p>
+      <PromptSpeech prompt={"miniscule"} />
```

Comment:
ya like here if you change the previous one to be ({ prompt }), it will just do prompt="miniscule." (u don't need to change anything here hehe)